src/i22_bluesky/stubs/linkam.py

```diff
@@ -1,17 +1,23 @@
+from typing import List
+
 import bluesky.plan_stubs as bps
 import numpy as np
-from dls_bluesky_core.core import group_uuid, step_to_num
-from dls_bluesky_core.stubs.flyables import fly_and_collect
+from dodal.common.coordination import group_uuid
+from dodal.common.maths import step_to_num
+from dodal.common.types import MsgGenerator
 from dodal.devices.linkam3 import Linkam3
 from ophyd_async.core.flyer import HardwareTriggeredFlyable
 
+from i22_bluesky.common.types import CollectableFlyable
 from i22_bluesky.panda.fly_scanning import RepeatedTrigger
+from i22_bluesky.stubs.fly import fly_and_collect
 
 
 # TODO: Make non-flyer signature to allow non-flying with non-flying device?
 def scan_linkam(
+    flyer: HardwareTriggeredFlyable[RepeatedTrigger],
+    devices: List[CollectableFlyable],
```

Comment:
panda, detectors as with the working fly_count?
```suggestion
    panda: PandA,
    detectors: List[CollectableFlyable],
```